sympy/matrices/tests/test_commonmatrix.py

```diff
@@ -466,19 +466,6 @@ def test_conjugate():
                           [ 5, 0]])
 
 
-def test_doit():
-    a = OperationsOnlyMatrix([[Add(x, x, evaluate=False)]])
-    assert a[0] != 2*x
-    assert a.doit() == Matrix([[2*x]])
-
-
-def test_evalf():
-    a = OperationsOnlyMatrix(2, 1, [sqrt(5), 6])
-    assert all(a.evalf()[i] == a[i].evalf() for i in range(2))
-    assert all(a.evalf(2)[i] == a[i].evalf(2) for i in range(2))
-    assert all(a.n(2)[i] == a[i].n(2) for i in range(2))
```

Comment:
Tests should not be removed without a valid reason.

Reply:
I am sorry for that. It was by mistake and was to be a part of issue #18666. I would revert those two commits soon.